src/sentry/consumers/__init__.py

```diff
@@ -396,12 +409,44 @@ def build_consumer_config(group_id: str):
             healthcheck_file_path, strategy_factory
         )
 
+    if enable_dlq:
+        try:
+            dlq_topic = consumer_definition["dlq_topic"]
+        except KeyError:
+            raise click.BadParameter(
+                f"Cannot enable DLQ for consumer: {consumer_name}, DLQ topic is not specified in consumer definition"
+            )
+        try:
+            cluster_setting = get_topic_definition(dlq_topic)["cluster"]
+        except ValueError:
+            raise click.BadParameter(
+                f"Invalid DLQ topic: {dlq_topic}, all DLQ topics must have topic definition configured in settings"
+            )
+
+        producer_config = get_kafka_producer_cluster_options(cluster_setting)
+        producer_config.pop("message.max.bytes", None)
```

Comment:
Is this needed?

Reply:
The other PRs I am referencing all remove them
https://github.com/getsentry/sentry/pull/53019
https://github.com/getsentry/sentry/pull/49537
https://github.com/getsentry/sentry/pull/54647

It's removed now